src/savegame_json.cpp

```diff
@@ -2733,3 +2734,78 @@ void deserialize( recipe_subset &value, JsonIn &jsin )
         value.include( &recipe_id( jsin.get_string() ).obj() );
     }
 }
+
+// basecamp
+void basecamp::serialize( JsonOut &json ) const
+{
+    json.start_object();
+    json.member( "name", name );
+    json.member( "x", pos.x );
+    json.member( "y", pos.y );
+    json.member( "z", pos.z );
+    json.member( "bb_x", bb_pos.x );
+    json.member( "bb_y", bb_pos.y );
+    if( !sort_points.empty() ) {
```

Comment:
The emptiness check seems unnecessary. The loading code handles an empty array just fine, and there won't be that many basecamps that it causes a significant increase in file size. But it will make the code simpler.

Reply:
Fixed.